builders/deploy.yaml

```diff
@@ -1,80 +1,112 @@
-options:
-  env:
-  - 'KO_DOCKER_REPO=us.gcr.io/${PROJECT_ID}'
-  - 'DOCKER_REPO_OVERRIDE=us.gcr.io/${PROJECT_ID}'
-steps:
-# Build ko, prerequisite.
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '--tag=gcr.io/$PROJECT_ID/ko', '-f', 'terraform/ko.Dockerfile', 'terraform/']
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['push', 'gcr.io/$PROJECT_ID/ko']
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-# Tests
-- name: 'mirror.gcr.io/library/golang'
-  env:
-  - GO111MODULE=on
-  args: ['go', 'test', './...']
-# Build and publish containers`
-- name: 'gcr.io/$PROJECT_ID/ko'
-  args:
-  - publish
-  - -P
-  - ./cmd/export
-- name: 'gcr.io/$PROJECT_ID/ko'
-  args:
-  - publish
-  - -P
-  - ./cmd/federationin
-- name: 'gcr.io/$PROJECT_ID/ko'
-  args:
-  - publish
-  - -P
-  - ./cmd/federationout
-- name: 'gcr.io/$PROJECT_ID/ko'
-  args:
-  - publish
-  - -P
-  - ./cmd/exposure
-- name: 'gcr.io/$PROJECT_ID/ko'
+#
+# Re-deploys existing Cloud Run services. The services must have been deployed
+# with Terraform first, before using this script.
+#
+
+substitutions:
+  _REGION:
+
+steps:
+- id: export
+  name: gcr.io/google.com/cloudsdktool/cloud-sdk:293.0.0-alpine
   args:
-  - publish
-  - -P
-  - ./cmd/cleanup-export
-- name: 'gcr.io/$PROJECT_ID/ko'
+  - 'bash'
+  - '-c'
+  - |-
+    gcloud run deploy export \
+      --quiet \
+      --project "${PROJECT_ID}" \
+      --platform "managed" \
+      --region "${_REGION}" \
+      --image "us.gcr.io/$PROJECT_ID/github.com/google/exposure-notifications-server/cmd/export:latest" \
+      --no-traffic
```

Comment:
I think we should leverage traffic splitting here and roll out to 5% on the deploy

Reply:
The `gcloud run deploy` command only supports "all" or "no" traffic. There's no way to specify rollout.

Reply:
that’s too bad... because it’s in the API :)

we’d have to make that 2 commands then

Reply:
I'd rather not duplicate promotion effort in deploy.yaml, since we already have promote.yaml. How about I update `./scripts/build` to call promote with 5% after?

Reply:
Actually, I think this should be two separate commands. That gives the ability to deploy a canary and change promotions independently.

Reply:
You could always run `./scripts/deploy && ./scripts/promote`, but I don't think we should clobber deployment and promotion.

Reply:
+1, that's why there is no percentage in deploy, it's OK to type 2 commands :)